### DISCUSSION TITLE
Guidance needed on analyzing non-KO, taxonomy-based OTU table data

### DISCUSSION THREAD

**Discussion by MizaR108** [cafferychen777/ggpicrust2#87] [Q&A] [answered]
Dear Caffery Yang,

I am quite new to microbiome analysis and have recently started exploring the potential of PICRUSt2 for my research. While I am enthusiastic about the possibilities this tool offers, I must admit that I am a bit lost when it comes to the specifics of using it with my data, which is different from the example KO data provided. I apologize for my novice level in this field and greatly appreciate any guidance you can provide.

My data consists of an OTU table with taxonomic annotations but does not include KEGG Ortholog (KO) identifiers. From your documentation and example data, I understand that the typical input for PICRUSt2 is a KO abundance table. However, I am uncertain about the procedure for analyzing data like mine, which is solely based on taxonomic information. My questions are as follows:

Preparing Taxonomy-Based OTU Table for PICRUSt2: What are the essential steps to prepare a taxonomy-based OTU table for PICRUSt2 analysis? Does the tool automatically infer the functional profiles (KO assignments) from the taxonomic data, or do I need to perform additional steps before running the analysis? 

Importance of Taxonomic Detail: How critical is the level of detail in the taxonomic annotations for the OTU table? For example, if some of my OTUs are annotated only to the family or genus level, will this significantly impact the accuracy or resolution of the functional predictions made by PICRUSt2?

Beginner-Friendly Resources or Tutorials: Are there any beginner-friendly resources or step-by-step tutorials that you would recommend for someone new to this type of analysis? Any materials that specifically address the transition from taxonomy-based OTU tables to functional predictions would be particularly helpful.

I am eager to learn and utilize PICRUSt2 effectively, but I find myself in need of some foundational guidance to ensure I am using the tool correctly for my type of data. Thank you very much for your support and for providing such a powerful resource to the scientific community.

Sincerely,

Joohwan

**Comment by cafferychen777** [comment, marked as answer]
Dear Joohwan,

Thank you for reaching out with your inquiries about using PICRUSt2 for your microbiome analysis.

Regarding your first question on converting OTU to KO using PICRUSt2, you are indeed correct that this is a necessary step. To assist you in this process, I recommend exploring the [PICRUSt2 GitHub repository](https://github.com/picrust/picrust2). This repository contains comprehensive information and guidelines that can help you in preparing your OTU table for PICRUSt2 analysis.

As for your second question about the impact of taxonomic detail level on the accuracy of functional predictions, my opinion is that it's unlikely to significantly affect the results. While having detailed taxonomic annotations can be beneficial, PICRUSt2 is designed to handle varying levels of taxonomic detail. Therefore, even if some of your OTUs are annotated only to the family or genus level, it should still be possible to obtain meaningful functional predictions.

Regarding your third question about beginner-friendly resources, especially those related to transitioning from taxonomy-based OTU tables to functional predictions, the PICRUSt2 wiki is an excellent place to start. It provides a step-by-step guide and useful information for beginners. Additionally, for a broader understanding of microbiome analysis, I highly recommend visiting [MicrobiomeStat Wiki](https://www.microbiomestat.wiki/). This website is a valuable resource for understanding various aspects of microbiome analysis, offering effective and practical information.

I hope these resources will be helpful in your research journey. Wishing you the best of luck with your work, and feel free to reach out if you have any more questions.

Sincerely,
Caffery Yang

**Reply by MizaR108** [reply]
Dear Caffery Yang,

Thank you very much for your quick and helpful response to my queries.

I am grateful for your support and the time you took to address my questions thoroughly. 

Warm regards,

Joohwan
